sdk/js/src/Variable.ts

```diff
@@ -34,6 +36,7 @@ export class DVCVariable<T extends DVCVariableValue> implements Variable<T> {
 
         this.defaultValue = variable.defaultValue
         this.eval = variable.eval
+        this._feature = variable._feature || undefined
```

Comment:
why not set it as null? that way you can just explicitly accept the value in the PB stuff?

Reply:
idk if this is related to PB, since that used SDKVariable type and that is the same type in the BucketedUserConfig. I can change it to null without any issues tho